02week/ticTacToe.js

```diff
@@ -37,23 +37,23 @@ function diagonalWin() {
 
 function checkForWin() {
   if (horizontalWin()) {
-    printBoard();
+    printBoard();  // refreshes the screen and shows the last play prior to the message below.
     console.log(`Congratulations player ${playerTurn}.  You notched a horizontal win`);
     return true;
   } else if (verticalWin()) {
-    printBoard();
+    printBoard();  // refreshes the screen and shows the last play prior to the message below.
     console.log(`Congratulations player ${playerTurn}.  You notched a vertical win`);
     return true;
   } else if (diagonalWin()) {
-    printBoard();
+    printBoard();  // refreshes the screen and shows the last play prior to the message below.
     console.log(`Congratulations player ${playerTurn}.  You notched a diagonal win`);
     return true;
   }
   return false;
 }
 
 function checkForTie() {
-  return board.every(square => square.trim() !== '');
+  return board[0].every(square => square.trim() !== '') && board[1].every(square => square.trim() !== '') && board[2].every(square => square.trim() !== '');
```

Comment:
Nice use of the `.every()` js method but could you increment a `moveCount`  variable and check the count here? This will prevent your app from having to loop through all your board logic twice every turn.

Hint: Only 9 valid moves can be made in the game before a tie is inevitable.

Ex: 
```
\\ code

board[row][column] = playerTurn;
moveCount++
```